examples/jsm/loaders/IFCLoader.js

```diff
@@ -363,7 +363,7 @@ class SubsetManager {
 
 	isValid( item ) {
 
-		return item != undefined && item != null;
+		return item !== undefined && item !== null;
```

Comment:
This should be fixed in the original repository.

cc @agviegas

Reply:
Indeed. Seems like the loader in the repo is a bit old now too.

Reply:
Hey, that's right! The project is growing a lot, and chances are that next year it will take off. To avoid bloating three.js and to have a single source of truth, maybe it's a better idea to keep it outside of three.js, like three-mesh-bvh? 🤔 What do you think?

Reply:
I think that's best - the three.js examples are still a good way to enable people to discover the external projects, though. The ifc js project can be modified to use an import map like the three-mesh-bvh raycasting example here:

https://github.com/mrdoob/three.js/blob/309b00afb6dcbc5e6c58e72f10eaa8d2e8888c83/examples/webgl_raycaster_bvh.html#L31-L39

Reply:
Ok! Should I update the example?

Reply:
That's my opinion - then we can delete the ifc.js files from the project and direct users to use the canonical repo.